client/src/ui/track/JobCard.tsx

```diff
@@ -69,16 +77,21 @@ export function JobCard({ application, profile }: JobCardProps) {
           padding: "4px",
         }}
       >
-        <Image
-          src="/landingpage/logo.svg"
-          height={40}
-          width={40}
-          alt="Squeakpeek Logo"
-          style={{
-            objectFit: "cover",
-            borderRadius: "8px",
-          }}
-        />
+        {logoUrl ? (
```

Comment:
I don't think check is necessary? logoUrl is always a type string

Reply:
yeah you are right!